### DISCUSSION TITLE
Challenge #2: Anticipating Household Displacement in Communities

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#2] [Challenges]
# Anticipating Household Displacement in Communities

Low-income renters face [displacement](https://sites.utexas.edu/gentrificationproject/understanding-gentrification-and-displacement/) (the set of demands that force households to move from their neighborhood of choice) at higher rates than others. Support to keep households in place–or support them in finding affordable housing–is needed. There are many datasets out there, but few places to analyze the data together, and fewer places still where there’s an ability for a renter to have the information to take action. 

In this challenge, you’ll work to understand the potential for displacement (as identified by cost-burden), integrate different data sources to create an understanding of displacement (including information on eviction filings), and prototype tools that give agency to a renter to find, apply for, and obtain housing.

* The big question a housing stakeholder is trying to answer with the results of this challenge is: where are communities vulnerable to displacement and how can organizations proactively address this potential displacement?  
* The big question a community resident is trying to answer is: how can I take action to ensure I have access to affordable housing?

## Community focus: Los Angeles

Los Angeles is a city with a historic (pre-pandemic) [high displacement risk](https://www.urbandisplacement.org/maps/los-angeles-gentrification-and-displacement/) in many communities, and numerous housing actors are [working](https://www.cakex.org/case-studies/climate-informed-anti-displacement-initiatives-los-angeles-enterprise-community-partners) to keep households in safe and affordable housing in their desired neighborhoods. These initiatives could benefit from increased learning and information on housing justice. [Chris Chenet](https://www.datakind.org/2023/11/07/welcome-chris-chenet-datakinds-product-owner/), working alongside collaborators at the [Homeless Policy Research Institute](https://hpri.usc.edu/), [Enterprise Community](https://www.enterprisecommunity.org/), and multiple actors from the Los Angeles Continuum of Care are looking to create a more current understanding of displacement risk so as to inform activities to increase agency for householders. 

In addition to addressing displacement, climate migration is becoming an increasingly significant concern. [Recent research](https://news.ucsc.edu/2024/07/housing-crisis-climate-risk.html#:~:text=In%20a%20new%20article%20for,near%20wildlands%2C%20leading%20to%20more) is exploring the impact of environmental and [climate factors](https://www.pnas.org/doi/10.1073/pnas.2310080121) on displacement, providing a more holistic view of the risks faced by communities. Chris and the broader collaborative team will be available throughout the DataKit process to engage with participants, answer questions, and offer expertise on housing justice and climate-related challenges. This initiative seeks to foster a future where households have greater agency and resilience in navigating these complex issues.

## Get started with existing data

[Grab the data here](https://github.com/datakind/datakit-housing-fall-2024/releases/latest).

Work to identify Census tracts where households are cost-burdened or severely cost burdened. Per the Dept of Housing and Urban Development (HUD), households that are cost burdened are those “who pay more than 30 percent of their income for housing” and households that are severely cost-burdened are those “paying more than 50 percent of one’s income on rent.”

Using the available data for Median Monthly Housing Costs and Median Household Income, begin to create an understanding of cost burden at a local level. (Tip: make sure you’ve normalized the two datasets, so that you’re either comparing monthly or annual finances to each other.) Consider evaluating this by Race (& Ethnicity), Gender and Age of Householder to inform insights around housing needs by population segments as well as income segments. If possible, also break down the analysis by families with minors, single individuals, and Transition-Age Youth (TAY), as most Continuums of Care (CoCs) typically use these categories in their assessments. DataKind has [pulled the data](https://github.com/datakind/datakit-housing-fall-2024/releases/latest) already for two case study states: California and Florida, and we encourage you to explore additional geographies as well through the [EODatascape](https://eodatascape.datakind.org). 

This “getting started” analysis should help answer the following questions

* What is the geographic distribution of cost burdened households? Are these tracts clustered in certain counties or regions?  
* What is the percentage of Census tracts that appear to be rent burdened?  
* Are there differences between the populations that are cost-burdened and those that are not?  
* What are some limitations of this tract-level analysis?  
* Are there differences between the populations that are severely cost-burdened and those that are not?  
* Reflect: What surprised you from this analysis? What are some limitations of the analysis? What are ways to extend the work?

## Take it further

The “Getting started with existing data” analysis is helpful for understanding the target community, and to take it further, analysis will center on finding and assessing potential drivers of displacement and policies to keep households in their communities.

* Research: how are other communities assessing the risk of displacement and responding to needs? What datasets are available to answer these questions?   
* Data analysis and visualization:   
  * Explore eviction filing rates in areas of cost-burden and severe cost-burden. Do eviction filings seem to occur at higher rates in these geographies? How quickly do landlords move from issuing a lock-out order or 3-day notice to filing an eviction? Are these landlords serial eviction filers for the same properties? Analyze whether serial eviction filings cluster around specific properties or areas, particularly in geographies with higher rates of cost-burdened households.  
  * Explore rental cost trends by census tract and identify areas where rental costs have increased the most over a selected time interval. Do these increases align with areas of high cost-burden, and are they correlated with recent zoning changes or city council decisions?  
  * Incorporate HUD’s Small Area Fair Market Rent (FMR) rates and Area Median Income (AMI) to provide deeper insights into the affordability challenges faced by low-income renters.  
  * Investigate complaints or civil rights violations filed by tenants against landlords. Which landlords or properties receive the most complaints, and are these properties located in areas classified as affordable? Cross-reference complaint data with eviction trends and displacement risks.  
* Prototyping:   
  * Create proposals for analysis, tools, or storytelling that answers the question of how low-income renters facing displacement can connect with affordable housing options within their communities.  
  * Consider an application for low-income renters to identify local affordable housing, request rental assistance, or file a complaint about landlord practices. What would the user flow look like? What information would need to be collected? What would need to be surfaced?

## Additional data sets to explore

* You can draw eviction filing rates and additional socio-economic data from the [Eviction Tracking System FEAT analysis for cities in Florida](https://www.featapp.org/eviction-data) or directly from the [Eviction Lab](https://evictionlab.org/map/). (Note: new census tracts were published in 2020\. You will need to use a [Census tract relationship file](https://www.census.gov/geographies/reference-files/time-series/geo/relationship-files.2020.html) to compare information between vintages.)  
* You can draw median rent from a number of sources including the [US Census table B25064](https://data.census.gov/table/ACSDT5Y2022.B25064?q=B25064:%20MEDIAN%20GROSS%20RENT%20\(DOLLARS\)&g=160XX00US4159000). (Note: new census tracts were published in 2020\. You will need to use a [Census tract relationship file](https://www.census.gov/geographies/reference-files/time-series/geo/relationship-files.2020.html) to compare information between vintages.)

**Comment by sana1410** [comment]
I have taken data from Los Angeles County and used Alteryx tool for data cleaning and performing data analysis. I have used Tableau for vislualisation. Below is the summary from my analysis.

1. What is the percentage of Census tracts that appear to be rent burdened?
![image](https://github.com/user-attachments/assets/a4152b9d-69ea-42a2-a7e0-65e67eea224b)
* Around 60% of census tracts are cost-burdened .

2. What is the geographic distribution of cost burdened households? Are these tracts clustered in certain counties or regions?
![image](https://github.com/user-attachments/assets/13a0c8dd-1b45-49a9-95c4-b31d97c276b1)

3. Are there differences between the populations that are cost-burdened and those that are not?
![image](https://github.com/user-attachments/assets/f397a462-ebc3-44f2-a3da-d837436a92ee)
More white only population are present in heavily cost burdened regions that cost burdened regions . 

4.Are there differences between the populations that are severely cost-burdened and those that are not?
![image](https://github.com/user-attachments/assets/42dd5ae0-ea84-40c5-a374-c41076b80dac)
Files :-
[tableau_outpout_LA_county.csv](https://github.com/user-attachments/files/17282506/tableau_outpout_LA_county.csv)
[summary.xlsx](https://github.com/user-attachments/files/17282514/summary.xlsx)
